alibi/datasets.py

```diff
@@ -50,13 +51,13 @@ def load_cats(target_size: tuple = (299, 299), return_X_y: bool = False) -> Unio
     (data, target)
         Tuple if ``return_X_y`` is true
     """
-    tar = tarfile.open(fileobj=BytesIO(pkgutil.get_data(__name__, "data/cats.tar.gz")), mode='r:gz')
+    tar = tarfile.open(fileobj=BytesIO(pkgutil.get_data(__name__, "data/cats.tar.gz")), mode='r:gz')  # type: ignore
```

Comment:
The three `type: ignore` in this module are because the various objects actually return `Optional`, presumably because the data is not guaranteed to exist or yield anything. I don't think we need to check for this explicitly as we are processing data that we ship with the library.